GitUI/CommandsDialogs/FormCleanupRepository.cs

```diff
@@ -78,17 +90,32 @@ private CleanMode GetCleanMode()
             throw new NotSupportedException($"Unknown value for {nameof(CleanMode)}.");
         }
 
-        private string? GetPathArgumentFromGui()
+        private string? GetInclusivePathArgumentFromGui()
         {
-            if (!checkBoxPathFilter.Checked)
+            if (!checkBoxIncludePathFilter.Checked)
             {
                 return null;
             }
 
             // 1. get all lines from text box which are not empty
             // 2. wrap lines with ""
             // 3. join together with space as separator
-            return string.Join(" ", textBoxPaths.Lines.Where(a => !string.IsNullOrEmpty(a)).Select(a => string.Format("\"{0}\"", a)));
+            return string.Join(" ", textBoxIncludePaths.Lines.Where(a => !string.IsNullOrEmpty(a)).Select(a => string.Format("\"{0}\"", a)));
+        }
+
+        private string? GetExclusivePathArgumentFromGui()
+        {
+            if (!checkBoxExcludePathFilter.Checked)
+            {
+                return null;
+            }
+
+            // 1. get all lines from text box which are not empty
+            // 2. Prepend lines with '-e '
+            // 3. Replace whitespace with '?' and convert to POSIX path
+            // 3. join together with space as separator
+
+            return string.Join(" ", textBoxExcludePaths.Lines.Where(a => !string.IsNullOrEmpty(a)).Select(a => string.Format("-e {0}", a.Replace(" ", "?")).ToPosixPath()));
```

Comment:
Please use the more meaningful `--exclude` (short option format should be used only by humans when typing the command from command line)

I also would have preferred a better letter than 'a'. 'p' for 'path'?

And maybe use string interpolation instead of string.Format()?